agent/06_agent_create_cluster.sh

```diff
@@ -499,6 +545,23 @@ case "${AGENT_E2E_TEST_BOOT_MODE}" in
     # (to avoid storage overconsumption on the CI machine)
     sudo rm -f "${OCP_DIR}/appliance.raw"
     ;;
+  "ISO_NO_REGISTRY" )
+    # Build an (OVE) image which does not need registry setup 
+    # Run a script from agent-installer-utils which internally uses openshift-appliance
+    asset_dir=$SCRIPTDIR/$OCP_DIR/iso_builder
+    mkdir -p ${asset_dir}
+    create_agent_iso_no_registry ${asset_dir}
+
+    attach_agent_iso_no_registry master $NUM_MASTERS
+    attach_agent_iso_no_registry worker $NUM_WORKERS
+
+    # In case of SNO, wait for additional 3 minutes for the node
+    # to finish booting up and make sure UI is up
+    if [[ "${NUM_MASTERS}" == "1" ]]; then
+      sleep 180
```

Comment:
Why this additional requirement?

Reply:
Right, its not needed. `check_assisted_install_UI` wait for the UI to be available anyway.